Frontend/ui-driver/src/Helpers/Utils.js

```diff
@@ -514,11 +514,11 @@ export const getPopupType = function (just, nothing) {
 export const renderSlider = function (cb) {
   return function (action) {
     return function (config) {
-      const { id, sliderConversionRate, sliderMinValue, sliderMaxValue, sliderDefaultValue, toolTipId } = config;
+      const { id, stepFunctionForCoinConversion, sliderConversionRate, sliderMinValue, sliderMaxValue, sliderDefaultValue, toolTipId } = config;
       const callback = callbackMapper.map(function (val) {
         cb(action(parseInt(val)))();
       });
-      window.JBridge.renderSlider(id, callback, sliderConversionRate, sliderMinValue, sliderMaxValue, sliderDefaultValue, toolTipId);
```

Comment:
we need to maintain backward compatibility here, by having a try catch block, call function with a parameter less in catch block

Reply:
Okay bro.